packages/backend-core/src/cache/tests/docWritethrough.spec.ts

```diff
@@ -10,15 +15,14 @@ import {
   init,
 } from "../docWritethrough"
 
-import InMemoryQueue from "../../queue/inMemoryQueue"
-
 const initialTime = Date.now()
 
 async function waitForQueueCompletion() {
-  const queue: InMemoryQueue = DocWritethroughProcessor.queue as never
-  await queue.waitForCompletion()
+  await utils.queue.processMessages(DocWritethroughProcessor.queue)
 }
 
+beforeAll(() => utils.queue.useRealQueues())
```

Comment:
Would it be worth treating `redis` like we treat `minio` and `couchdb`, adding them to `globalSetup.ts` and making sure they're always running during tests?

Reply:
I started this way, but we might have conflicts sharing redis instances as all the tests might affect each other...

Reply:
Ahhh yes, I am remembering now that I also did try this. Redis doesn't have great ways to isolate tests away from each other like the DBs do (where we create a new database every time).